app/apple/server/appleOauthRegisterService.ts

```diff
@@ -0,0 +1,76 @@
+import { KJUR } from 'jsrsasign';
+import { ServiceConfiguration } from 'meteor/service-configuration';
+
+import { CustomOAuth } from '../../custom-oauth/server/custom_oauth_server';
+import { settings, settingsRegistry } from '../../settings/server';
+
+const config = {
+	serverURL: 'https://appleid.apple.com',
+	tokenPath: '/auth/token',
+	scope: 'name email',
+	mergeUsers: true,
+	accessTokenParam: 'access_token',
+	loginStyle: 'redirect',
```

Comment:
should we use 'popup' instead? I remember we discussing how 'redirect' could be bad for the desktop app.

Reply:
I dont think this has any effect since we trigger the action by the apple script
but sure its a little messed